clio-client/src/test/scala/org/broadinstitute/clio/client/commands/ClioParsersSpec.scala

```diff
@@ -18,9 +18,13 @@ class ClioParsersSpec extends FlatSpec with Matchers with ClioParsers {
   DocumentStatus.values.foreach {
     it should behave like enumParser(_)
   }
+  DataType.values.foreach {
+    it should behave like enumParser(_)
```

Comment:
Do we need to test every enum separately here? E.g. is there something special that needs to be done for each enum for this to work, or would it be sufficient to test a single enum?